VideoLocker/res/layout/row_discussion_comment.xml

```diff
@@ -16,32 +17,37 @@
         android:textSize="@dimen/edx_x_small"
         tools:text="This is the text of a comment. This is the text of a comment. This is the text of a comment. This is the text of a comment. This is the text of a comment." />
 
-    <TextView
-        android:id="@+id/discussion_comment_count_report_text_view"
-        android:layout_width="wrap_content"
+    <LinearLayout
+        android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_alignParentRight="true"
-        android:layout_alignParentEnd="true"
-        android:layout_below="@id/discussion_comment_body"
-        android:background="?android:selectableItemBackground"
-        android:clickable="true"
-        android:drawablePadding="5dp"
-        android:padding="@dimen/discussion_responses_box_padding"
-        android:textColor="@color/edx_grayscale_neutral_base"
-        android:textSize="@dimen/edx_xxx_small"
-        tools:text="@string/discussion_responses_report_label" />
+        android:gravity="center"
+        android:orientation="horizontal">
 
-    <TextView
-        android:id="@+id/discussion_comment_author_text_view"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_alignLeft="@id/discussion_comment_body"
-        android:layout_alignStart="@id/discussion_comment_body"
-        android:layout_below="@id/discussion_comment_body"
-        android:layout_marginTop="@dimen/discussion_responses_box_padding"
-        android:layout_toLeftOf="@id/discussion_comment_count_report_text_view"
-        android:layout_toStartOf="@id/discussion_comment_count_report_text_view"
-        android:textColor="@color/edx_grayscale_neutral_base"
-        android:textSize="@dimen/edx_xxx_small"
-        tools:text="Brian 10 minutes ago" />
-</RelativeLayout>
\ No newline at end of file
+        <TextView
+            android:id="@+id/discussion_comment_author_text_view"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:padding="@dimen/discussion_responses_box_padding"
+            android:textColor="@color/edx_grayscale_neutral_base"
+            android:textSize="@dimen/edx_xxx_small"
+            tools:text="Brian 10 minutes ago" />
+
+        <android.support.v4.widget.Space
```

Comment:
We could get rid of this `Space` if we assigned a weight of 1 to one of the `TextView` items instead of using `wrap_content` :wink: